packages/appeals-service-api/src/db/schema.prisma

```diff
@@ -1,5 +1,6 @@
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
+  binaryTargets = ["native", "darwin"]
```

Comment:
I also had to add this a while ago but my prisma has been well behaved without it for a while. Do we definitively need this?

Reply:
I'll take it out